advec1d.py

```diff
@@ -0,0 +1,97 @@
+'''
+Copyright (C) 2017-2019  Waterloo Quantitative Consulting Group, Inc.
+See COPYING and LICENSE files at project root for more details.
+'''
+
+import numpy as np
+from lib import pyblitzdg
+import matplotlib.pyplot as plt
+
+def advec1dComputeRHS(u,c, nodes1d):
+    vmapM = nodes1d.vmapM
+    vmapP = nodes1d.vmapP
+    mapI = nodes1d.mapI
+    mapO = nodes1d.mapO
+    nx = nodes1d.nx
+    rx = nodes1d.rx
+    Dr = nodes1d.Dr
+    Lift = nodes1d.Lift
+    Fscale = nodes1d.Fscale
+
+    uC = np.squeeze(np.reshape(u, (u.shape[0]*u.shape[1],1), order='F'), axis=1)
+    nxC = np.squeeze(np.reshape(nx, (nx.shape[0]*nx.shape[1],1), order='F'), axis=1)
+    RHS = 0*vmapM
```

Comment:
RHS = np.zeros(vmapM.shape) ?

Reply:
Sure, I'll change. Wasn't sure what was the preferred way to write it when doing numpy things.

Reply:
Actually this line isn't even needed, RHS is replaced at line 35